config/servicemappings/accesscontextmanager.yaml

```diff
@@ -356,3 +356,30 @@ spec:
                 group: resourcemanager.cnrm.cloud.google.com
               targetField: "number"
               valueTemplate: "projects/{{value}}"
+    - name: google_access_context_manager_service_perimeter_resource
+      kind: AccessContextManagerServicePerimeterResource
+      idTemplate: "{{perimeter_name}}/{{resource}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      v1alpha1ToV1beta1: true
+      storageVersion: v1alpha1
+      autoGenerated: false
+      resourceReferences:
+        - key: projectRef
+          tfField: resource
+          description: |-
+            A GCP resource that is inside of the service perimeter.
+          gvk:
+            kind: Project
+            version: v1beta1
+            group: resourcemanager.cnrm.cloud.google.com
+          targetField: number
+          valueTemplate: "projects/{{value}}"
+        - key: perimeterNameRef
```

Comment:
Could you add doc to clarify this is an external only reference field. See example here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/certificatemanager.yaml#L61